plugins/default/lib/mongodb_health.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import traceback
+from collections import Counter
+
+import openshift
+import nagios
+from mongodb import ReplsetStatuses
+
+
+def generate_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-s', '--selector', type=str, action="store",
```

Comment:
`type=str` and `action="store"` are the defaults, no need to write that :)

I would suggest the "template" from #23:

```python
def generate_parser():
    parser = argparse.ArgumentParser(
        description="Checks the status of a MongoDB replica set",
    )
    parser.add_argument(
        "-s", "--selector", required=True,
        help='label selector for MongoDB pods (e.g.: "name=mongodb")',
    )
    return parser
```

The `required=True` argument is perhaps not "required", it could be okay to filter by no selector (e.g. passing an empty selector == not passing any). The nagios command always pass the argument anyway.


I'm also in doubt about taking a "selector" as input, instead of the service name, more on future comments :)